### DISCUSSION TITLE
I Can't Open Osu Lazer Anymore

### DISCUSSION THREAD

**Discussion by OkinagaHoshimi** [ppy/osu#28289] [Q&A] [answered]
Hey, so I've sent an email to the support email two days ago and they told me to report this here

So i've been using osu lazer for a few weeks now, and so far i can use it no problem. However, when i downloaded the newest update (May 19th 2024), at first it started to lag really bad. I thought it was because the app is starting to get too heavy for my laptop so i tried to mess with the graphics.

Changing the resolution doesn't help, so i tried to mess with the renderer available in the options. The app then told me to restart the game. However when i tried to open the game, the app won't open.

I thought maybe changing the renderer might broke the app somehow, so i deleted the app and reinstalled it, but the app still won't open. 

I'm using windows 10 and had the compatibility mode turned off already so i don't think it's a compatibility issue. I can't send you any images or logs from the game because well, the app won't open at all. Do you have any idea what might be the problem and is there any way for me to fix it?

**Comment by cihe13375** [comment, marked as answer]
Try deleting `framework.ini`. It's in `%userprofile%\AppData\Roaming\osu` (paste this string into the address bar of File Explorer and press enter).

Reinstalling the game does not reset the settings so it did not solve your issue.

**Reply by OkinagaHoshimi** [reply]
It worked, thank you
